lib/exercises.rb

```diff
@@ -1,29 +1,112 @@
-
 # This method will return an array of arrays.
 # Each subarray will have strings which are anagrams of each other
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: o(n2) because I am iterating over the string and
+# I know Bytes is gonna iterate over it as well
+# Space Complexity: 0(n)
 
 def grouped_anagrams(strings)
```

Comment:
👍 The time complexity is O(n) since you're iterating over a list of words and you can assume a small size for each word.